pages/techgenius/tg-planet-user-manual.md

```diff
@@ -243,53 +243,53 @@ Fill out the following form:
 
 ![](images/tg-course-add-details.png)
 
-1. **Course Title.**  The name or title of the Course.
+1. **Course Title.** The name or title of the Course.
 
-2. **Language of Instruction.**  The language of Course materials. Please use the name of the language in that language. For example, العربية .
+2. **Language of Instruction.** The language of Course materials. Please use the name of the language in that language. For example, العربية .
  
-3. **Member Limit.**  The number of Learners that are allowed to enter the Course.
+3. **Member Limit.** The number of Learners that are allowed to enter the Course.
 
-4. **Course Leader.**  Choose a single Leader or multiple Leaders to manage the Course
+4. **Course Leader.** Choose a single Leader or multiple Leaders to manage the Course
 
-5. **Description.**  Provide a course description.
+5. **Description.** Provide a course description.
 
-6. **Method.**  Describe the way you envision the Course being used (e.g., school Course, enrichment Course, training manual, independent learning, special programs, etc.)
+6. **Method.** Describe the way you envision the Course being used (e.g., school Course, enrichment Course, training manual, independent learning, special programs, etc.)
 
-7. **Grade Level.**  Suggested grade level of the Learners
+7. **Grade Level.** Suggested grade level of the Learners
 
-8. **Subject Level.**  Subject level - beginner, intermediate, or advanced. 
+8. **Subject Level.** Subject level - beginner, intermediate, or advanced. 
 
-9. **Start Date.**  Start date of the Course (only if applicable)
+9. **Start Date.** Start date of the Course (only if applicable)
 
-10. **End Date.**  End date of the Course (only if applicable)
+10. **End Date.** End date of the Course (only if applicable)
 
-11. **Frequency.**  Frequency of Course meeting, either Daily or Weekly (only if applicable)
+11. **Frequency.** Frequency of Course meeting, either Daily or Weekly (only if applicable)
 
-12.	**Start Time.**  Start time of the Course (only if applicable)
+12.	**Start Time.** Start time of the Course (only if applicable)
```

Comment:
Can you also remove space here :)

Reply:
@lmmrssa It is actually single-spaced when I previewed it on the rawgit. It is kind of weird that we could see a double space here. Here I attach the screenshot of it

![image](https://user-images.githubusercontent.com/25775298/42840138-7c4fd880-89cb-11e8-809c-240baf27c2a6.png)